### DISCUSSION TITLE
Filebeat 3rd party module dashboards

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#5596] [Unsupported Versions] [answered]
Hello.
I need help with 3rd party module: I want to add [threat intel module](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-threatintel.html), so I've edited pillar's config, restarted filebeat, but I can't see any dashboards from it.
Did someone try to install this module? Or dashboards from 3rd party modules not working in SO at all?
Thanks!

**Comment by dougburks** [comment, marked as answer]
We don't really support Filebeat modules automatically loading their dashboards into Kibana right now. You may be able to manually export the dashboard from the module and then import into Kibana.

**Reply by SuiNom** [reply]
Hey Folks,

Just curious if the solution here worked, I downloaded Filebeat and found the dashboard json files:
```
➜  dashboard pwd
/home/SuiNom/Downloads/filebeat-7.15.0-linux-x86_64/kibana/7/dashboard
➜  dashboard grep -i o365 *
Filebeat-O365-Audit.json:        "title": "[Filebeat o365] Audit Dashboard ECS",
➜  dashboard 
```

However when I try to import the dashboard via `Kibana > Stack Management > Saved Objects > Import` I get the following error:
```
Sorry, there was an error
Saved objects file format is invalid and cannot be imported.
```
![2021-09-29_11-57_1](https://user-images.githubusercontent.com/22012477/135255961-3020a826-0ce7-4b5b-86dc-e1102cd83551.png) ![2021-09-29_11-57](https://user-images.githubusercontent.com/22012477/135255966-a32e4d3c-0510-4f98-a17c-4fd10b517817.png)

Would appreciate any help if anyone has these working.

Cheers!

**Reply by dougburks** [reply]
@SuiNom Make sure that you download the version of Filebeat that matches the version of the Elastic stack.  If you have further questions or problems, please start a separate discussion. Thanks!
